integration_test/third_party_apps_test.go

```diff
@@ -187,6 +190,34 @@ type expectedEntry struct {
 	FieldMatchers map[string]string `yaml:"field_matchers"`
 }
 
+// expectedMetricsEntry encodes a series of assertions about what data we expect
```

Comment:
Typo: your type is actually called `expectedMetricEntry` (no `s`). However, why not just call this `expectedMetric`?

Reply:
`expectedMetric` works but I'd be tempted to go ahead and rename the existing `expectedEntry` to `expectedLog` as well, if @martijnvans is OK with that.

Reply:
yep sure am OK with it.

Reply:
Fixed.